Fixes #5194 - Add Wolverhampton (EGBO) Heli pads and Update SMR

RedstonePilot opened the pull request:
Fixes #5194

# Summary of changes

Added 6 new Heli Pads
Added 4 Heli aiming points
Added HAP
Added Ops Buildings
Added disused areas where appropriate (according to the chart)
Changed the south helipads to be hard surfaced
Changed runway dimensions (as per the charts)
Updated KMZ file

# Screenshots

![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/b3f444f1-7ea8-4a2c-a226-6d9d845429b1)

![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/4316198a-7092-4dbe-ae6b-7a3447353180)

PLM1995 reviewed (commented):
Looks good! Some minor points:

The area near C/D should have taxiways onto 10/28 according to the chart, might be worth depicting that?
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/41bf24b8-f467-4350-966a-c47c757deeb4)

My initial suggestion was for the helipads on the apron, to use just a `H` label, I personally find the hand-drawn "H" in a circle a bit hard to view at normal zooms. Do others have a view on this?
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/61326713/97e4d178-3631-4b4f-9d4a-988179ace440)

Thanks @RedstonePilot

RedstonePilot commented:
> The area near C/D should have taxiways onto 10/28 according to the chart, might be worth depicting that?

Will add those

RedstonePilot commented:
Added the taxiways at C and D. Also added this grass route to the end of 28 
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/93eb8a36-1248-4823-94f7-1e90d1aea860)


Changed H to labels but not committed. I do agree they look _much_ better
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/ef098692-63d8-4c94-8df0-b9c24cc22aaa)

So if you are happy will commit that change

RedstonePilot commented:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/142dec89-7350-47c4-9bb2-f54f74ebf5ab)

This is marked as asphalt parking now. Should it be left as is or changed to a grey?

PLM1995 commented:
-------

> Changed H to labels but not yet pushed. I do agree they look _much_ better ![image](https://private-user-images.githubusercontent.com/146777065/306272100-d9a218c6-2a8e-4036-b7e9-6c9733ba9030.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0Mzk1MzcsIm5iZiI6MTcwODQzOTIzNywicGF0aCI6Ii8xNDY3NzcwNjUvMzA2MjcyMTAwLWQ5YTIxOGM2LTJhOGUtNDAzNi1iN2U5LTZjOTczM2JhOTAzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMFQxNDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02MGUyOTE5YzcxNWM0NDJjZTZlMzVmMDQwZTk4NjNlMGQ5MzdhZTRmNmVhYWRjMDk2MWUzMDdkMmMzYzk1NzEwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.PV3Xi82hwSzkff8tr2yn9yvzBjBkBAeIdo0ssLG7JuQ)
> 
> So if you are happy will push that change

How does that look at 'normal' zoom - i.e. when viewing the whole airfield? Does the circle interfere with the label? I was thinking just have a `H` label and nothing else.

-------

> ![image](https://private-user-images.githubusercontent.com/146777065/306275145-142dec89-7350-47c4-9bb2-f54f74ebf5ab.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDg0Mzk1MzcsIm5iZiI6MTcwODQzOTIzNywicGF0aCI6Ii8xNDY3NzcwNjUvMzA2Mjc1MTQ1LTE0MmRlYzg5LTczNTAtNDdjNC05YmIyLWY1NGY3NGViZjVhYi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjIwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIyMFQxNDI3MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lNTQ4NTg1Mjg2MWJiOTllNTM1OWZjZjE5MDVjZjc2NWE4NjU3YWU5MzMzMjZlZjExMmY1YmJiZjEzZmM3NmJiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.z4BejnKtCbChDjWdJgUj9ObHMU2Xeb1KeAtuc6nZz9s)
> 
> This is marked as asphalt parking now. Should it be left as is or changed to a grey?

I don't think it matters too much as it's distinct, but as it is non-grass, feel free to change the colour if you wish.

-------

C/D bits looks good, ta. Just remember to upload the latest .kmz once everything is sorted :)

RedstonePilot commented:
Kmz is updated. I'll check the H and remove the circle as appropriate. From memory it begins to clip around the normal view

RedstonePilot commented:
![image](https://github.com/VATSIM-UK/UK-Sector-File/assets/146777065/7357e2d3-7b61-4e00-aa90-5264156fa7fe)


The circles did indeed overlap

PLM1995 reviewed (approved):
Looks good!

AliceFord reviewed (approved):
Looks good to me, nice job 😄
